nav_msgs/Odometry Odometry
team2_package/vehicle_state vehicle_state
sensor_msgs/NavSatFix NavSatFix